lib/capistrano/db_sync/postgres/cli.rb

```diff
@@ -4,11 +4,11 @@ def initialize(config)
   end
 
   def dump(to_file, db, options = [])
-    "#{with_pw} pg_dump #{credentials} #{format_args} -f #{to_file} #{options.join(' ')} #{db}"
+    "#{with_pw} pg_dump #{credentials} #{format_args} -f #{to_file} #{options.join(' ')} #{db}".strip
```

Comment:
I've added a `String#strip` to remove the leading space when `with_pw` is empty.

returning a proper Capistrano `Command` object instead of a string might be a better option. I've explored it yet.